test/methods.js

```diff
@@ -1270,6 +1270,30 @@ test("postalcodeBR", function() {
 	ok( !method( "99.999999"), "Invalid BR Postal Code");
 });
 
+test("nit_pis_pasepBR", function() {
+	var method = methodTest("nit_pis_pasepBR");
+	ok( method( "125.6659.879-9"), "Valid NIT/PIS/Pasep"); //Number automatically generated
+	ok( method( "125.9189.997-7"), "Valid NIT/PIS/Pasep");
+	ok( method( "125.3815.082-7"), "Valid NIT/PIS/Pasep");
+	ok( method( "125.5145.923-2"), "Valid NIT/PIS/Pasep");
+	ok( method( "125.2138.841-8"), "Valid NIT/PIS/Pasep");
+	ok( method( "12566598799"), "Valid NIT/PIS/Pasep");
+	ok( !method( "125.6659.879-8"), "Invalid NIT/PIS/Pasep");
+	ok( !method( "12566598798"), "Invalid NIT/PIS/Pasep");
+});
+
+test("teBR", function() {
+	var method = methodTest("teBR");
+	ok( method( "4261 6271 1546"), "Valid TE"); //Number automatically generated
+	ok( method( "407113120361"), "Valid TE");
+	ok( method( "6218 7807 0531"), "Valid TE");
+	ok( method( "0352 6016 1554"), "Valid TE");
+	ok( method( "352601615/54"), "Valid TE");
+	ok( method( "35260161554"), "Valid TE");
+	ok( !method( "6238 7807 0531"), "Invalid TE");
+	ok( !method( "421878070531"), "Invalid TE");
```

Comment:
that's why we do this code review. This project follows the [jQuery Code Style](http://contribute.jquery.org/style-guide/js/) documented on the [contributing](https://github.com/jzaefferer/jquery-validation/blob/master/CONTRIBUTING.md) file.

It might seem strict, but it's necessary for the sake of maintainability. If you need help, I can specify each point. The missing points are basically the space between arguments and parenthesis, e.g.:  

```diff
- ok( !method( "421878070531"), "Invalid TE");
+ ok( !method( "421878070531" ), "Invalid TE" );
```

Also the comments should be placed on the live above, with another blank line above, with a single space after the double slashes. E.g.:

```diff
- var method = methodTest("teBR");
- ok( method( "4261 6271 1546"), "Valid TE"); //Number automatically generated
+ var method = methodTest( "teBR" );
+
+ // Number automatically generated
+ ok( method( "4261 6271 1546" ), "Valid TE" );
```